aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -1056,7 +1056,7 @@ Tensor tensor_cpu(ArrayRef<T> values, const TensorOptions& options) {
 
 template <typename T>
 Tensor tensor_backend(ArrayRef<T> values, const TensorOptions& options) {
-  auto cpu_tensor = tensor_cpu(values, options.device(DeviceType::CPU));
+  auto cpu_tensor = tensor_cpu(values, options.device(kCPU));
```

Comment:
Just some cleanup- these two are aliased and I saw that we use the `k*` variant in most places.